README.md

```diff
@@ -61,6 +61,8 @@ OpenShift LightSpeed (OLS) is an AI powered assistant that runs on OpenShift and
    points to file containing a valid OpenAI, Watsonx etc. API key, and change the `default_model` and `default_provider` values to
    reference the selected provider and model.
 
+   Note: there are two supported methods to provide credentials for Azure OpenAI. The first method is compatible with other providers, i.e. `credentials_path` contains directory name containing one file with API token. In the second method, that directory should contain three files named `tenant_id`, `client_id`, and `client_secret`. Please look at following articles describing how to retrieve this information from Azure: https://learn.microsoft.com/en-us/azure/azure-portal/get-subscription-tenant-id https://azurelessons.com/how-to-get-client-id-and-client-secret-in-azure-portal/
```

Comment:
I would probably add an "and" in between the links just to have some separation between them and make it clear they're two separate links.

Reply:
or use a list of title with hyperlink